### ISSUE TITLE
Critère 12.10: condition 2 trop restrictive par rapport aux WCAG

### ISSUE THREAD

**Issue by OlivierNourry** [DISIC/RGAA#71]
> Un mécanisme est disponible pour configurer la touche de raccourci clavier au moyen des touches de modification (Ctrl, Alt, Maj, etc).

- "reconfigurer" aurait été plus proche du terme d'origine "to remap"
- Cette condition, telle qu'elle est rédigée, sous-entend que l'ajout d'un modificateur est le seul moyen de reconfigurer les touches concernées. Or le doc "How to meet" du W3C indique:

> The remapping mechanism _**includes**_ non-printing characters. The alternative shortcuts _**could be longer strings**_ of up to 25 characters that would directly serve as native speech commands for any speech engine.

(emphase ajoutée)

S'il faut donc effectivement introduire un modificateur, ce n'est pas restreint à cela; il est aussi possible d'ajouter des caractères imprimables supplémentaires. Il me paraît nécessaire de le préciser, pour rester fidèle aux WCAG.

**Comment by alixraison** [comment]
Merci pour votre signalement.

La remarque du caractère restrictif de l’énoncé de la condition 2 du test 12.10.1 n’est pas recevable dans la mesure où cet énoncé s’aligne sur le même niveau d’exigence que celui de la seconde modalité d’application du SC 2.1.4.

Cette exigence établit la nécessité de pouvoir définir un raccourci qui inclut un ou plusieurs modificateurs. Cependant, un raccourci ne peut pas reposer que sur des caractères de modification. Ces caractères de modification doivent être suivis au moins d’un autre caractère. C’est ce que précise le « how to meet » : possibilité d’ajouter un énoncé simple en tant que raccourci après le modificateur; cette possibilité est laissée à la discrétion de l’utilisateur si le mécanisme permet ce type de configuration.

Ajout au critère 12.10  de deux techniques WCAG récemment ajoutées et associées au SC 2.1.4 :

- G127 (https://www.w3.org/WAI/WCAG21/Techniques/general/G217) ;
- F99 (https://www.w3.org/WAI/WCAG21/Techniques/failures/F99).
